UpdateVersions.ps1

```diff
@@ -0,0 +1,40 @@
+Param(
+    [string]$WinAppSDKVersion = "",
```

Comment:
Where are we getting this variable from?

Reply:
From the caller :)

Reply:
Sorry, I missed the %* in the cmd file

Reply:
Specifically, if the caller was our nightly build, the nightly build will have the version variable that it used for packing the nuget stored somewhere